client/lib/plans/features-list.js

```diff
@@ -1036,7 +1036,7 @@ export const FEATURES_LIST = {
 
 	[ FEATURE_SCAN_V2 ]: {
 		getSlug: () => FEATURE_SCAN_V2,
-		getTitle: () => i18n.translate( 'Automated daily scanning' ),
+		getTitle: () => i18n.translate( 'Real-time threat scanning' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'Automated real-time scanning' )` *ES Score: 6*